specification/network/resource-manager/Microsoft.Network/stable/2021-03-01/virtualWan.json

```diff
@@ -5448,6 +5448,240 @@
           "final-state-via": "location"
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/virtualHubs/{virtualHubName}/routingIntent/{routingIntentName}": {
+      "put": {
+        "operationId": "RoutingIntent_CreateOrUpdate",
+        "x-ms-examples": {
+          "RouteTablePut": {
+            "$ref": "./examples/RoutingIntentPut.json"
+          }
+        },
+        "description": "Creates a RoutingIntent resource if it doesn't exist else updates the existing RoutingIntent.",
+        "parameters": [
+          {
+            "$ref": "./network.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The resource group name of the RoutingIntent."
+          },
+          {
+            "name": "virtualHubName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the VirtualHub."
+          },
+          {
+            "name": "routingIntentName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the Routing Intent resource."
```

Comment:
should we describe it as "The name of the per VirtualHub singleton Routing Intent resource" here?